data/scripts/dependency-resolver.py

```diff
@@ -285,8 +306,10 @@ def handle_filesystem_check(args, conf, context):
     if found_path:
         context.add_kconfig("HAVE_%s" % dep.upper(), "bool", "y")
         context.add_cond_makefile_var("%s_PATH" % dep.upper(), found_path)
+        context.add_log_content("%s found" % dep)
     else:
         context.add_kconfig("HAVE_%s" % dep.upper(), "bool", "n")
+        context.add_log_content("%s found" % dep)
 
```

Comment:
Ok, I just didn't include the kconfig key:value pairs just because it's written to Kconfig.gen file any ways...